### PR TITLE
[PAY-1283] DMs: Listen for chat messages app wide, and get unread message count on load

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Thanks @schottra for the subcomponent idea

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Tested against stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-dms-app-wide

**Review by dharit-tan — APPROVED** [review]

File: packages/web/src/components/chat-listener/ChatListener.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const { connect, disconnect, fetchMoreChats, fetchUnreadMessagesCount } =
+  chatActions
+const { getAccountStatus } = accountSelectors
+
+export const ChatListener = () => {
+  const dispatch = useDispatch()
+  const accountStatus = useSelector(getAccountStatus)
+  // Connect to chats websockets and prefetch chats
+  useEffect(() => {
+    if (accountStatus === Status.SUCCESS) {
```
Comment (piazzatron) [inline packages/web/src/components/chat-listener/ChatListener.tsx:15]: Any risk on dispatching these actions multiple times in the case of sign out + sign in again in same session?

Comment (rickyrombo) [reply packages/web/src/components/chat-listener/ChatListener.tsx:15]: Yes, but it disconnects in the effect cleanup

Comment (piazzatron) [reply packages/web/src/components/chat-listener/ChatListener.tsx:15]: duh

File: packages/web/src/components/chat-listener/ChatListener.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      dispatch(connect())
```
Comment (schottra) [inline packages/web/src/components/chat-listener/ChatListener.tsx:16]: I'm assuming it's safe for these three actions to run simultaneously without waiting for any to complete?

Comment (rickyrombo) [reply packages/web/src/components/chat-listener/ChatListener.tsx:16]: Yep!

**Pull request merged**